src/main.rs

```diff
@@ -81,7 +81,9 @@ fn startup(settings: &settings::SpotSettings) {
         .set_gtk_application_prefer_dark_theme(settings.prefers_dark_theme);
 
     //Make the slider move on click
-    gtk::Settings::default().unwrap().set_gtk_primary_button_warps_slider(true);
+    gtk::Settings::default()
```

Comment:
Is this sufficient to make the slider work? Based on what I understand this will only change it for the application.

Reply:
right, but i'd rather leave that to its default value so the slider acts consitently with other apps (and the user's preferences)

Reply:
Alright fine but maybe in the readme include this just in case a user may not know that. I for a while had just assumed that it was not implemented.

Reply:
sure, go ahead :)